model-engine/model_engine_server/common/dtos/llms.py

```diff
@@ -565,6 +573,7 @@ def from_api(request: CreateBatchCompletionsRequest) -> "CreateBatchCompletionsE
             output_data_path=request.output_data_path,
             content=request.content,
             model_config=request.model_config,
+            model_cfg=request.model_config,
```

Comment:
is there a reason why we have model_config and model_cfg?

Reply:
it's kinda jank, but it's for forward/backwards compatibility.

On the API side, we want to keep model_config, but on the engine side, we want model_cfg